### PR TITLE
Feat/order

### LINKED ISSUE #4: Creazione Entity Ordine

### PR DESCRIPTION
**Pull request by Fabio013**
Create entity Order e model OrderDTO

### CODE REVIEW

File: src/main/java/com/develhope/Java27_progetto3_team2/entity/Order.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    private Long id;
+
+    // Stato corrente dell'ordine (Esempi: Pending, In Preparation, Delivering, Completed ecc.)
+    @Setter
+    @Column(name = "status", nullable = false)
+    private String status; //Temporaneo in attesa che un enumerator degli status venga implementato
+
+    // Indirizzo di consegna fornito dal cliente
+    @Setter
+    @Column(name = "deliveryAddress", nullable = false)
```
Comment (DonatoAletta) [inline src/main/java/com/develhope/Java27_progetto3_team2/entity/Order.java]: The @Column indicates the name of the table on the db which usually isn't in camel case. change the name to delivery_address

File: src/main/java/com/develhope/Java27_progetto3_team2/entity/Order.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    private String deliveryAddress;
+
+    // Metodo di pagamento scelto dal cliente (Esempi: Carta di credito, PayPal)
+    @Setter
+    @Column(name = "paymentMethod", nullable = false)
```
Comment (DonatoAletta) [inline src/main/java/com/develhope/Java27_progetto3_team2/entity/Order.java]: same here, change to payment_method

File: src/main/java/com/develhope/Java27_progetto3_team2/entity/Order.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    private String paymentMethod;
+
+    // Data di creazione dell'ordine
+    @Setter
+    @Column(name = "orderDate", nullable = false)
```
Comment (DonatoAletta) [inline src/main/java/com/develhope/Java27_progetto3_team2/entity/Order.java]: Same, order_date

File: src/main/java/com/develhope/Java27_progetto3_team2/entity/Order.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    private Date orderDate;
+
+    // Prezzo totale dell'ordine
+    @Setter
+    @Column(name = "totalPrice", nullable = false)
```
Comment (DonatoAletta) [inline src/main/java/com/develhope/Java27_progetto3_team2/entity/Order.java]: Same, total_price

File: src/main/java/com/develhope/Java27_progetto3_team2/entity/Order.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    private Double totalPrice;
+
+    // ID del corriere assegnato all'ordine
+    @Setter
+    @Column(name = "courierId", nullable = false)
```
Comment (DonatoAletta) [inline src/main/java/com/develhope/Java27_progetto3_team2/entity/Order.java]: Sa,e courier_id

**Review by DonatoAletta — CHANGES_REQUESTED** [review]
Check all the comments and change the @Column names

**Comment by Fabio013** [comment]
Names of columns renamed as request to fit the database formatting @DonatoAletta

**Review by DonatoAletta — APPROVED** [review]

**Review by AleCam04 — APPROVED** [review]

**Pull request merged**